alpha/scripts/utils/migrationWebcastUpdateMakeHiddenAndDisplayInSearch.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+require_once(__DIR__ . '/../bootstrap.php');
+
+if (count($argv) < 2)
+{
+    die('pleas provide partner id as input' . PHP_EOL .
+        'to run script: ' . basename(__FILE__) . ' X' . PHP_EOL .
+        'whereas X is the path to the partner ids file' . PHP_EOL);
+}
+
+$dryRun = !(count($argv) == 3 && @$argv[2] == "realrun");
+if ($dryRun)
+{
+    KalturaLog::debug('this is a dry run. pass "realrun" as 2nd param to make it actually do stuff');
+}
+
+$partners = file(@$argv[1]);
```

Comment:
Use something like this for fetching the file:
$entriesFilePath = $argv[1];
$entries = file ( $entriesFilePath ) or die ( 'Could not read file [$entriesFilePath]!' );